packages/stream-metadata/package.json

```diff
@@ -9,7 +9,10 @@
 		"build": "node ./esbuild.config.mjs",
 		"lint": "eslint . --ext .ts",
 		"lint:fix": "eslint . --ext .ts --fix",
-		"dev": "yarn build && node ./dist/node_esbuild.cjs",
+		"dev": "yarn dev:local_multi",
+		"dev:local_multi": "yarn run-build",
+		"dev:local_multi_ne": "RIVER_ENV=local_multi_ne PORT=3003 yarn run-build",
+		"run-build": "dotenv -e .env.local yarn build && node ./dist/node_esbuild.cjs",
```

Comment:
@tak-hntlabs dotenv should have no impact here. you can remove everything before `yarn build`. we should have no env dependencies at compile time. if you want your runtime to depend on .env files, you should handle that in the source code.

i'm saying this while assuming that you actually don't have any explicit dependencies to the environment at build time. if you do, we should remove them. this isn't frontend code like vite, so we can build once and run in multiple environments without having to rebuild

Reply:
tested and removed dotenv-cli